docs/tutorials/RunWithAutowareArchitectureProposal.md

```diff
@@ -0,0 +1,87 @@
+```bash
+Would you like to install the optional GPU modules (cuda: 11.1, cudnn: 8.0.5, TensorRT: 7.2.1)? (y/n) [y]: 
+```
+
+| NOTE: This step will most likely modify your existing display drivers setup which might disrupt GUI system layer. If you already have proprietary display driver installed, it would be recommended to remove it first |
+|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+
+Confirm if you wish this script to install those components and wait for it to finish. 
+With GPU related installation it might take up to 45 minutes to complete.
+
+4. Build workspace:
+```bash
+colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+```
+
+## How to run
+
+To run scenario use following command, which should execute `autoware-simple.yaml` scenario from `scenario_test_runner` package.
+```bash
+source install/setup.bash
+ros2 launch scenario_test_runner scenario_test_runner.launch.py scenario:=src/simulator/scenario_simulator/test_runner/scenario_test_runner/test/scenario/autoware-simple.yaml vehicle_model:=lexus sensor_model:=aip_xx1
+```
+
+It will execute simple scenario where ego moves some distance along the road.
+
+### Expected behavior
+
+During sample scenario following things are expected to happen:
+1. `rviz` is started with dedicated view. 
+2. Lanelet map shows up
+3. Ego spawns in one of the lanes
+4. Ego receives a goal and plan the route - local trajectory is visualized in `rviz` as a green bar
+5. Ego starts driving and moves along the lane to the goal
+6. Ego stops and scenario ends with information about successful execution:
+
+```
+[openscenario_interpreter_node-3] [INFO] [1636493579.969013627] [simulation.openscenario_interpreter]: Deactivating.
+[openscenario_interpreter_node-3] [INFO] [1636493580.123393080] [simulation.concealer]: Shutting down Autoware: (1/3) Stop publlishing/subscribing.
+[openscenario_interpreter_node-3] [INFO] [1636493580.123491540] [simulation.concealer]: Shutting down Autoware: (1/3) Stoped publlishing/subscribing.
+[openscenario_interpreter_node-3] [INFO] [1636493580.123604379] [simulation.concealer]: Shutting down Autoware: (2/3) Send SIGINT to Autoware launch process.
+[openscenario_interpreter_node-3] [INFO] [1636493580.123679047] [simulation.concealer]: Shutting down Autoware: (2/3) Terminating Autoware.
+...
+[openscenario_interpreter_node-3] [INFO] [1636493583.702040114] [simulation.concealer]: Shutting down Autoware: (3/3) Waiting for Autoware to be exited.
+[openscenario_interpreter_node-3] [INFO] [1636493583.778138473] [simulation.openscenario_interpreter]: Passed
+```
+
+During scenario execution you should be able to see `rviz` view similar to the one below:
+
+![scenario_simualtor_with_aap](../image/scenario_simulator_with_autoware_architecture_proposal.png)
+
```

Comment:
You can add that the chosen scenario repeats 3 times.

Reply:
I guess it could be added as point 7. in the list above